cove_ocds/management/commands/ocds_cli.py

```diff
@@ -30,10 +30,23 @@ def handle(self, file, *args, **options):
             ))
 
         try:
-            result = ocds_json_output(self.output_dir, file, schema_version, convert)
+            if os.path.isdir(file):
+                directory = os.fsencode(file)
+                outfile = open(os.path.join(self.output_dir, "all_results.txt"), 'w+')
```

Comment:
Use `with open` syntax, otherwise if there's an exception, `outfile.close()` will not be reached.